akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -15,16 +15,15 @@ To use Akka Cluster, you must add the following dependency in your project:
 ## When and where to use Akka Cluster
 
 An architectural choice you have to make is if you are going to use a microservices architecture or
-a traditional distributed application, also known as a distributed monolith. This choice will influence how
-you should use Akka Cluster
+a traditional distributed application. This choice will influence how you should use Akka Cluster.
 
 ### Microservices
 
 Microservices has many attractive properties, such as the independent nature of microservices allows for
 multiple smaller and more focused teams that can deliver new functionality more frequently and can
 respond quicker to business opportunities. Reactive Microservices should be isolated, autonomous, and have
 a single responsibility as identified by Jonas Bonér in the book
-[Reactive Microservices Architecture: Design Principles for Distributed Systems](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html).
+[Reactive Microservices: The Evolution of Microservices at Scale](https://info.lightbend.com/ebook-reactive-microservices-the-evolution-of-microservices-at-scale-register.html).
```

Comment:
I know the URL says "reactive-microservices" but the actual title is "Reactive Microsystems"

Reply:
Doh, thanks.

Sun was before their time here also. :)